languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/prohibit.txt

```diff
@@ -8671,4 +8671,30 @@ Judenhunde/SN
 Judenhündin
 Judenhündinnen
 Hey Jude
-hey Jude
\ No newline at end of file
+hey Jude
+Vieresszimmer.*
+Fünferzdimmer.*
+Fünferdzimmer.*
+Fünferbzimmer.*
+Fünferzzimmer.*
+Sechserzdimmer.*
+Sechserdzimmer.*
+Sechserbzimmer.*
+Sechserzzimmer.*
+Siebenerzdimmer.*
+Siebenerdzimmer.*
+Siebenerbzimmer.*
+Siebenerzzimmer.*
+Neunerzdimmer.*
+Neunerdzimmer.*
+Neunerbzimmer.*
+Neunerzzimmer.*
+Zehnerzdimmer.*
+Zehnerdzimmer.*
+Zehnerbzimmer.*
+Zehnerzzimmer.*
+Erkerzimmer/NS
```

Comment:
Das ist aber ein korrektes Wort und sollte daher nicht in der `prohibit.txt` erscheinen, oder?

Reply:
Erkerzimmer hatte ich noch nie gehört, wird entfnernt.